pkgs/applications/window-managers/picom/picom-allusive.nix

```diff
@@ -19,7 +19,7 @@ picom.overrideAttrs (oldAttrs: rec {
   '' + (lib.optionalString (oldAttrs ? postInstall) oldAttrs.postInstall);
 
   meta = (builtins.removeAttrs oldAttrs.meta [ "longDescription" ]) // {
-    description = "A fork of picom featuring improved animations and other features";
+    description = "The only Picom fork you will ever need. With Animations and More.";
```

Comment:
desc shouldn't end in a period
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description